shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -668,7 +668,7 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
       result.setImportantForAccessibility(isImportant(semanticsNode));
     }
 
-    // Work around for https://github.com/flutter/flutter/issues/2101
```

Comment:
I believe this comment is still needed, but should be pushed down into the if/else block. Otherwise, LGTM! Thanks for helping improve the quality of our code comments.

Reply:
I didn't remove it, but changed the issue to the correct one. If you look at #2101 you'll notice it has nothing to do with the comment, while #22545 is the correct one.